src/content/docs/codestream/codestream-ui-overview/observability-section.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: CodeStream observability
+metaDescription: "The observability section is where you connect CodeStream to your New Relic account so that you can take advantage of these observability features."
```

Comment:
I'm not sure this is how I'd describe this section. That would be like describing the Pull Requests section by saying that it's how you connect to GitHub. Yes, we'll encourage you to connect if not already, but the purpose of the section is to show you errors.

Reply:
@planteater This is a fair comment. Happy to update. Not personally monitoring anything in New Relic, even though I was connected to New Relic, I never saw anything in the Observability section. It wasn't until maybe yesterday that I finally saw a screenshot of this :D There's a lot of moving parts.

I think I'd go with something like:

"Once you've connected CodeStream and New Relic, use the observability section to see your errors."